src/cms/templates/feedback/feedback_form.html

```diff
@@ -0,0 +1,106 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% block content %}
+{% load static %}
+{% load widget_tweaks %}
+<h2>Feedback</h2> 
+
+<table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white" border="1">
+<tr class="border-b border-solid border-gray-200">
+  <th>Type</th>
+  <th>Objekt</th>
+  <th>Comment</th>
+  <th>Rating</th>
+  <th>Date</th>
+</tr>
+
+
+{% for item in feedback_events %} 
+<tr><td align="center">Event</td>
+<td align="center">https://integreat.app/{{item.event.permalink}}</td>
+<td align="center">{{ item.comment }}</td>
+  {% if item.emotion == "NEG"  %}
+       <td align="center"> Down </td>
+  {% else %}
+      <td align="center" >Up</td>
+  {% endif %}
+  <td align="center">{{ item.created_date }}</td></tr>
+{% endfor %}
+{% for item in feedback_offers %}
+<tr><td align="center">Offer</td>
+<td align="center">{{item.offer.url}}</td>
+  <td align="center">{{ item.comment }}</td>>
+   {% if item.emotion == "NEG" %}
+       <td align="center"> Down </td>
+  {% else %}
+      <td align="center">Up</td>
+  {% endif %}<td align="center">{{ item.created_date }}</td></tr>
+{% endfor %}
+{% for item in feedback_regions %}
+<tr><td align="center">Region</td>
+  <td align="center">{{item.region.permalink}}</td>
+  <td align="center">{{ item.comment }}</td>
+  {% if item.emotion == "up"  %}
+    <td align="center"> up</td>
+  {% else %}
+     <td align="center">down</td>
+  {% endif %}<td align="center">{{ item.created_date }}</td></tr>
+{% endfor %}
+{% for item in feedback_pages %}
+<tr><td align="center">Pages</td>
+  <td align="center">{{ item.page.url }}</td>
```

Comment:
```suggestion
<td align="center">
  <a href="{% url 'edit_page' page_id=item.page_translation.page.id region_slug=region.slug language_code=item.page_translation.language.code%}">
    {{ item.page_translation.title }}
  </a>
</td>
```
Note: this change only works, if you change the `page` foreign key of `PageFeedback` to `page_translation` similar to what you did in `EventFeedback`. Also, it only works when you moved the feedback list to a region-specific list.